frameworks/hdfs/universe/resource.json

```diff
@@ -3,7 +3,8 @@
     "uris": {
       "jre-tar-gz": "{{jre-url}}",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
-      "hdfs-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.11.0.tar.gz",
+      "keytab-zip": "https://s3.amazonaws.com/hdfs-binary/krbtest-2.0.tar.gz",
+      "hdfs-tar-gz": "https://s3.amazonaws.com/hdfs-binary/hadoop-3.2.0.tar.gz",
```

Comment:
Since we know these tarballs are good, let's move them to `downloads.mesosphere` and update the links.

Reply:
+1

Reply:
Done